src/locales/en.json

```diff
@@ -168,11 +176,15 @@
   "link": "Add link",
   "image": "Add image",
   "get_boost": "Get the Busy boost!",
-  "get_boost_content": "Want to boost your post? Add the topic \"busy\" and get a like from us.",
+  "get_boost_content":
+    "Want to boost your post? Add the topic \"busy\" and get a like from us.",
   "drafts": "Drafts",
   "drafts_empty": "You don't have any draft saved",
   "draft_untitled": "Untitled draft",
   "draft_delete": "Delete this draft",
+  "draft_delete_success": "Draft have been deleted",
```

Comment:
It should be `Draft has been deleted`. We should change this here and in `defaultMessage` as well.